### PR TITLE
 Subscribe potential partners

### PR DESCRIPTION
**Pull request by sophieH29**

### CODE REVIEW

File: database.rules.json [hunk 1]
```diff
@@ -26,6 +26,10 @@
       ".read": false,
       ".write": true
     },
+    "potentialPartners": {
+      ".read": false,
+      ".write": true
```
Comment (unreadableusername) [inline database.rules.json]: it's possible to erase all data by anyone

Comment (sophieH29) [reply database.rules.json]: Anyone can write there. What you can propose?

Comment (unreadableusername) [reply database.rules.json]: you can disallow writing to the root of `potentialPartners`, but allow write to it's childs

Comment (sophieH29) [reply database.rules.json]: please give an example, and I'll change at other places

**Pull request merged**